roles/k3s/master/tasks/main.yml

```diff
@@ -193,3 +200,5 @@
     - "{{ k3s_server_manifests_directories.files }}"
   loop_control:
     label: "{{ item.path }}"
+
+
```

Comment:
Can we remove this?

Reply:
yep.  I'm not a pull request expert, still learning, so I don't know how to easily pull those back --- give me a few to get that right.  I have a few of these PRs as you can see, so I may have to detangle a few things.  THe last thing I did was delete the whole PR... and I don't want to do that this time.

out of curiosity - with respect to how you use the scripts, do you run ansible on the master node?  Do you always run kubectl from the masters themselves?  Just trying to understand the use-case of NOT pulling the master file...  I can delete it or I can setup a new variable too...

Reply:
> yep. I'm not a pull request expert, still learning, so I don't know how to easily pull those back 

No worries - instead great that you are learning new things 😊
You can "just" add other commits to the source branch (in this case `node-copy-kube-config`) to revise your proposed changes (e.g. delete those lines).

---

> out of curiosity - with respect to how you use the scripts, do you run ansible on the master node? Do you always run kubectl from the masters themselves? Just trying to understand the use-case of NOT pulling the master file... I can delete it or I can setup a new variable too...

I'm running Ansible on my laptop (which is not part of a cluster) and I'm (mostly) running `kubectl` from there. So if i need to obtain the `kubecfg` after creating a new cluster, I copy it over manually (which means `scp some-control-node:.kube/config /target/on/local/machine` instead of `cp /tmp/kube_config.some-vip /target/on/local/machine` with your patch).

More generally speaking, in my opinion setting up a computer to _manage_ the cluster from (i.e, installing `kubectl`, obtaining kubecfg etc.) is a separate task from setting up the cluster. So if I would be switching over from running `kubectl` manually to say keeping a repo of manifests to apply automatically via ansible, setting up the "manager" node would be a concern of different Ansible roles. This gives me (and everyone else) lots of flexibility and keeps things simple (at least in my opinion) ☺️